### PR TITLE
Make dry_run a setting and --force its inverse

### PR DESCRIPTION
**Pull request by mdellweg**
fixes TBA

### CODE REVIEW

**Review by daviddavis — APPROVED** [review]
LGTM. Maybe update https://github.com/pulp/pulp-cli/blob/develop/docs/install.md#katello ?

File: docs/install.md [hunk 1]
```diff
@@ -87,3 +87,6 @@ key = "/etc/pki/katello/private/pulp-client.key"
 verify_ssl = false
 ```
 
+As Katello uses Pulp as a backend all modifying actions in Pulp should be performed via Katello.
+Therefore you are also strongly encourged to set `dry_run = false`, to prevent accidentally calling into dangerous commands.
+This setting can in turn be overwritten on the command-line with the `--force` flag.
```
Comment (mdellweg) [inline docs/install.md:92]: @daviddavis 
Is this, what you had in mind?

**Review by daviddavis — APPROVED** [review]

File: docs/install.md [hunk 2]
```diff
@@ -87,3 +87,6 @@ key = "/etc/pki/katello/private/pulp-client.key"
 verify_ssl = false
 ```
 
+As Katello uses Pulp as a backend all modifying actions in Pulp should be performed via Katello.
+Therefore you are also strongly encourged to set `dry_run = false`, to prevent accidentally calling into dangerous commands.
```
Comment (evgeni) [inline docs/install.md:91]: Should that have been `dry_run = true`? ;-)

Comment (mdellweg) [reply docs/install.md:91]: absolutely!

**Pull request merged**